homeassistant/components/spc/binary_sensor.py

```diff
@@ -6,13 +6,14 @@
 from homeassistant.components.binary_sensor import (
     DEVICE_CLASS_MOTION,
     DEVICE_CLASS_OPENING,
+    DEVICE_CLASS_POWER,
     DEVICE_CLASS_SMOKE,
     BinarySensorEntity,
 )
 from homeassistant.core import callback
 from homeassistant.helpers.dispatcher import async_dispatcher_connect
 
-from . import DATA_API, SIGNAL_UPDATE_SENSOR
+from . import DOMAIN, SIGNAL_UPDATE_SENSOR
```

Comment:
DOMAIN in platforms are expected to represent the platform domain (binary_sensor)
```suggestion
from . import DOMAIN as SPC_DOMAIN, SIGNAL_UPDATE_ALARM
```

Reply:
Thanks. Fixed!